app/views/admin/deploy_groups/edit.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Edit Deploy Group</h1>
+
+<%= form_for [:admin, @deploy_group], html: { class: "form-horizontal" } do |form| %>
+  <%= render 'admin/deploy_groups/fields', form: form %>
```

Comment:
this is all identical to new except for the button label, so maybe put all into fields and either use "Save" for both or do `label = (@deploy_group.new_record? ? "Create" : "Save")`